x-pack/plugins/cloud_security_posture/public/pages/findings/utils.ts

```diff
@@ -20,3 +21,19 @@ export const getBaseQuery = ({
   // will be accounted for before releasing the feature
   query: buildEsQuery(dataView, query, filters),
 });
+
+export const getEuiPaginationFromEs = <T extends unknown>({
+  from: pageIndex,
+  size: pageSize,
+  total,
+}: {
+  total?: number | undefined;
+  size: number;
+  from: number;
+}): EuiBasicTableProps<T>['pagination'] => ({
+  pageSize,
+  pageIndex: Math.ceil(pageIndex / pageSize),
```

Comment:
I find it confusing, renaming `from` to `pageIndex` and later calculating `pageIndex` with `pageIndex`/ `pageSize`

Reply:
renamed back to `from`